### ISSUE TITLE
Document which settings the script expects on the printer

### ISSUE THREAD

**Issue by darix** [sedrubal/brother_printer_fwupd#13]
e.g. it expects SNMP to be enabled and configured with snmp v2c. A security focused person might set this to v3 or have the service off completely. which of the other services in network -> network -> protocol does it actually use for uploading the firmware file?

**Comment by sedrubal** [comment]
Yes, we should document the required protocols and the script should notify if one required protocol is not enabled. AFAIK we use

- zeroconf / mdns to find the printer which is optional
- SMTP v2c for querying the model information
- pdl-datastream (port 9100) to upload the firmware to the printer

Currently, I don't have access to a brother printer and can't check the naming of the protocols that is used by brother to write informative messages. Do you want to open a pull request?
